### PR TITLE
[INLONG-8423][Manager] Manager client support tenant operation

### LINKED ISSUE #8423: [Feature][Manager] Manager client support tenant operation
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by castorqin**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8423  

### Motivation
add two clients in mangaer client

- InlongTenant client 
- InlongTenantRole client

### Modifications
The details can be found at [#8423  ](https://github.com/apache/inlong/issues/8423).

### CODE REVIEW

File: inlong-manager/manager-client/pom.xml [hunk 1]
```diff
@@ -71,6 +71,12 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.pagehelper</groupId>
+            <artifactId>pagehelper</artifactId>
+            <version>5.3.0</version>
```
Comment (dockerzhang) [inline inlong-manager/manager-client/pom.xml:77]: Please move the version definition to the root pom.xml.

**Review by haifxu — APPROVED** [review]

File: inlong-manager/manager-client/pom.xml (hunk 1 above)
Comment (castorqin) [reply inlong-manager/manager-client/pom.xml]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+public class InlongTenantClient {
+
+    private final InlongTenantApi inlongTenantApi;
+
+    public InlongTenantClient(ClientConfiguration configuration) {
+        inlongTenantApi = ClientUtils.createRetrofit(configuration).create(InlongTenantApi.class);
+    }
+
+    /**
+     * get inlong tenant by tenant name
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java:41]: Get inlong tenant by tenant name

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+     * @return {@link InlongTenantInfo}
+     */
+    public InlongTenantInfo getTenantByName(String name) {
+        Response<InlongTenantInfo> inlongTenantInfoResponse = ClientUtils.executeHttpCall(inlongTenantApi.get(name));
+        ClientUtils.assertRespSuccess(inlongTenantInfoResponse);
+        return inlongTenantInfoResponse.getData();
+    }
+
+    /**
+     * create inlong tenant
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java:53]: Create inlong tenant

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+     * @return tenant id
+     */
+    public int save(TenantRoleRequest record) {
+        Response<Integer> saveResult = ClientUtils.executeHttpCall(inlongTenantRoleApi.save(record));
+        ClientUtils.assertRespSuccess(saveResult);
+        return saveResult.getData();
+    }
+
+    /**
+     * Update one tenant role
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:65]: Update tenant role

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+     * @return true/false
+     */
+    public boolean update(TenantRoleRequest record) {
+        Response<Boolean> updateResult = ClientUtils.executeHttpCall(inlongTenantRoleApi.update(record));
+        ClientUtils.assertRespSuccess(updateResult);
+        return updateResult.getData();
+    }
+
+    /**
+     * Get one tenant role by id
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:77]: Get tenant role by id

Comment (castorqin) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java (hunk 4 above)
Comment (castorqin) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java (hunk 3 above)
Comment (castorqin) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java]: done

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java (hunk 2 above)
Comment (castorqin) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantClient.java]: done

**Review by vernedeng — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    private final InlongTenantRoleApi inlongTenantRoleApi;
+
+    public InlongTenantRoleClient(ClientConfiguration configuration) {
+        this.inlongTenantRoleApi = ClientUtils.createRetrofit(configuration).create(InlongTenantRoleApi.class);
+    }
+
+    /**
+     * List all tenant role by paginating
+     *
+     *@param request tenant page info
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:41]: * @param request tenant page info

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+     *@return {@link PageInfo<TenantRoleInfo>}
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:42]: * @return {@link PageInfo<TenantRoleInfo>}

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+        ClientUtils.assertRespSuccess(pageInfoResponse);
+        return pageInfoResponse.getData();
+    }
+
+    /**
+     * Save  tenant role
+     *
+     * @param record tenant role info
+     *
+     * @return tenant id
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:56]: * @param record tenant role info
* @return tenant id

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+        this.inlongTenantRoleApi = ClientUtils.createRetrofit(configuration).create(InlongTenantRoleApi.class);
+    }
+
+    /**
+     * List all tenant role by paginating
+     *
+     * @param request tenant page info
+     * @return {@link PageInfo}
+     */
+    public PageInfo<TenantRoleInfo> listByCondition(TenantRolePageRequest request) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongTenantRoleClient.java:44]: Client not need the use of `PageHelper. PageInfo `, please change this interface to use the `PageResult` in manaber-web, thanks.

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**